components/TProfile.vue

```diff
@@ -302,6 +296,9 @@ import {
 import { useCities } from '~/use/cities'
 
 export default {
+  components: {
+    TOrderButton,
```

Comment:
Nuxt is doing auto-import of components. No need to define components, they are already available.